Scripts/UpdateInsightsTableCreationSettings.ps1

```diff
@@ -0,0 +1,58 @@
+<# 
+.SYNOPSIS 
+Script that enables/disables per process/queue table creation for UiPath Insights.
+#>
+param(
+    [Parameter(Mandatory=$true)][string]$server # Orchestrator endpoint
+    ,[Parameter(Mandatory=$true)][string]$tenancyName # Tenant name
+    ,[Parameter(Mandatory=$true)][string]$user # Tenant user name
+    ,[secureString]$password = $(Read-Host -Prompt "Enter password" -AsSecureString) # Tenant user password (prompted)
+    ,[boolean]$perProcessTablesEnabled = $true # Boolean to enable/disable process tables
+    ,[boolean]$perQueueTablesEnabled = $true # Boolean to enable/disable queue tables
+)
+
+$credentials = New-Object System.Management.Automation.PSCredential `
```

Comment:
Is the `System.Management.Automation.PSCredential`  already included, or do we need to import another dll

Reply:
We don't appear to, it looks like it's part of core.